packages/jasmine-auto-spies/src/tests/create-spy-from-class.spec.ts

```diff
@@ -125,7 +131,9 @@ describe('createSpyFromClass', () => {
 
   describe(`GIVEN a synchronous method is configured to throw on mismatch
             WHEN called with the wrong parameters`, () => {
+    let methodCalled: string;
```

Comment:
I would change it to `expectedMethodName` 

because people might confuse `methodCalled` with whether it has been called (invoked) or not